src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -76,15 +80,17 @@ public ErrorListInfoBarController(IHost host, IUnboundProjectFinder unboundProje
         #region IErrorListInfoBarController
         public void Reset()
         {
-            this.AssertOnUIThread();
+            threadHandling.ThrowIfNotOnUIThread();
 
             this.ClearCurrentInfoBar();
         }
 
         public void Refresh()
         {
+            logger.LogDebug("[ErrorListInfoBarController] In Refresh");
```

Comment:
I added a few debug statements to make it easier to see what was happening in the UI.